python/ray/train/examples/mlflow_simple_example.py

```diff
@@ -2,7 +2,7 @@
 from ray import train
 from ray.train import ScalingConfig, RunConfig
 from ray.train.torch import TorchTrainer
-from ray.air.integrations.mlflow import MLflowLoggerCallback
+from ray.tune.logger.mlflow import MLflowLoggerCallback
```

Comment:
More for a follow-up, but it's pretty annoying that we have to import from tune here

Reply:
Yeah agreed, though this does subclass `ray.tune.logger.LoggerCallback` so I think it's okay for now.